tabular/src/autogluon/tabular/learner/default_learner.py

```diff
@@ -32,12 +32,14 @@ def __init__(self, trainer_type=AutoTrainer, **kwargs):
         self._time_limit = None
 
     # TODO: Add trainer_kwargs to simplify parameter count and extensibility
-    def _fit(self, X: DataFrame, X_val: DataFrame = None, scheduler_options=None, hyperparameter_tune=False,
+
+    def fit(self, X: DataFrame, X_val: DataFrame = None, X_unlabeled: DataFrame = None,scheduler_options=None, hyperparameter_tune=False,
```

Comment:
bug: don't rename

nit: no space between new argument and scheduler_options

Reply:
Changed function name back and added a space.